omnibus/files/private-chef-cookbooks/private-chef/templates/default/oc_erchef.config.erb

```diff
@@ -145,7 +145,6 @@
             chef_<%= node['private_chef']['opscode-erchef']['search_provider'] %>
         ]},
         {base_resource_url, <%= @helper.erl_atom_or_string(node['private_chef']['opscode-erchef']['base_resource_url']) %>},
-        {bulk_fetch_batch_size, <%= @bulk_fetch_batch_size %>},
```

Comment:
Why is this being removed?

Reply:
There was some kind of error with bulk_fetch_batch_size being defined twice.  I have no idea how or when that happened, or why the issue just popped-up.  I can investigate reverting this change and see what happens.

Reply:
I see, it is here too: https://github.com/chef/chef-server/pull/2070/files#diff-a141dfd9bcf6d4f53e6f78764f42cd3bR85